lightning/src/chain/keysinterface.rs

```diff
@@ -498,7 +498,7 @@ impl ChannelKeys for InMemoryChannelKeys {
 		Ok(local_commitment_tx.get_local_sig(&self.funding_key, &channel_funding_redeemscript, self.channel_value_satoshis, secp_ctx))
 	}
 
-	#[cfg(test)]
+	#[cfg(any(test,feature = "unsafe"))]
```

Comment:
I presume this is for your own internal tests? Is there a way we could run those tests ourselves?

Reply:
We have several functional tests, which are very similar to the rust-lightning functional tests, but obviously use our signer.  When we exercise things like triggering a justice tx, we need to do unsafe things, like getting the local tx, but then move on to the next commitment state.

I'm not sure I understand what you mean by "run those tests ourselves".

Reply:
I'm not sure how to drag them without pulling the whole crate ?